<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncludeUi</class>
 <widget class="QDialog" name="IncludeUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="3">
      <widget class="QPushButton" name="browsePB">
       <property name="toolTip">
        <string>Select a file</string>
       </property>
       <property name="text">
        <string>&amp;Browse...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="TextLabel1">
       <property name="text">
        <string>I&amp;nclude Type:</string>
       </property>
       <property name="buddy">
        <cstring>typeCO</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="typeCO">
       <item>
        <property name="text">
         <string>Include</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Input</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Verbatim</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Program Listing</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="editPB">
       <property name="toolTip">
        <string>Edit the file</string>
       </property>
       <property name="text">
        <string>&amp;Edit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="filenameED">
       <property name="toolTip">
        <string>File name to include</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="filenameLA">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>&amp;File:</string>
       </property>
       <property name="buddy">
        <cstring>filenameED</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="visiblespaceCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Underline spaces in generated output</string>
       </property>
       <property name="text">
        <string>&amp;Mark spaces in output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="previewCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Show LaTeX preview</string>
       </property>
       <property name="text">
        <string>&amp;Show preview</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="listingsGB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Listing Parameters</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="labelLE"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLabel">
        <property name="text">
         <string>&amp;Label:</string>
        </property>
        <property name="buddy">
         <cstring>labelLE</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="listingsED">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QTextBrowser" name="listingsTB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="captionLabel">
        <property name="text">
         <string>&amp;Caption:</string>
        </property>
        <property name="buddy">
         <cstring>captionLE</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="captionLE">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="bypassCB">
        <property name="toolTip">
         <string>Check it to enter parameters that are not recognizable by LyX</string>
        </property>
        <property name="text">
         <string>&amp;Bypass validation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;More parameters</string>
        </property>
        <property name="buddy">
         <cstring>listingsED</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="literalCB">
        <property name="toolTip">
         <string>Pass content of the `Caption' field literally to LaTeX. Check this if you want to enter LaTeX code.</string>
        </property>
        <property name="text">
         <string>Li&amp;teral</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filenameED</tabstop>
  <tabstop>browsePB</tabstop>
  <tabstop>typeCO</tabstop>
  <tabstop>editPB</tabstop>
  <tabstop>visiblespaceCB</tabstop>
  <tabstop>previewCB</tabstop>
  <tabstop>captionLE</tabstop>
  <tabstop>labelLE</tabstop>
  <tabstop>listingsTB</tabstop>
  <tabstop>listingsED</tabstop>
 </tabstops>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
